apps/admin-web/jest.config.js

```diff
@@ -5,6 +5,9 @@ module.exports = {
     '^(?!.*\\.(js|jsx|ts|tsx|css|json)$)': '@nrwl/react/plugins/jest',
     '^.+\\.[tj]sx?$': 'babel-jest',
   },
+  moduleNameMapper: {
+    '^lodash-es$': 'lodash',
+  },
```

Comment:
This means jest will use lodash instead of lodash-es, this means dev might have a bigger size but prod won't. 
https://stackoverflow.com/a/54117206

`transformIgnorePatterns` did not work